packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/service.js

```diff
@@ -4,12 +4,41 @@ export default class OctaviaLoadBalancerHealthMonitorService {
     this.$http = $http;
   }
 
-  // WORK IN PROGRESS
-  // TODO: USE REAL API TO RETRIEVE THE POOL HEALTH MONITOR
-  // TODO: REMOVE ESLINT-DISABLE
-  /* eslint-disable class-methods-use-this */
-  /* eslint-disable no-unused-vars */
   getHealthMonitor(projectId, region, poolId) {
-    return [];
+    return this.$http
+      .get(
+        `/cloud/project/${projectId}/region/${region}/loadbalancing/healthMonitor`,
+      )
+      .then(({ data }) =>
+        data.filter((healthMonitor) => healthMonitor.poolId === poolId),
+      );
+  }
+
+  createHealthMonitor(projectId, region, poolId, healthMonitor) {
+    const postDatas = {
+      maxRetries: healthMonitor.maxRetries,
+      maxRetriesDown: healthMonitor.maxRetriesDown,
+      monitorType: healthMonitor.type,
+      name: healthMonitor.name,
+      periodicity: `PT${healthMonitor.periodicity}S`,
+      poolId,
+      timeout: healthMonitor.timeout,
+    };
+
+    if (healthMonitor.urlPath && healthMonitor.expectedCode) {
+      postDatas.httpConfiguration = {
+        expectedCodes: [healthMonitor.expectedCode],
+        httpMethod: 'GET',
+        httpVersion: 1.0,
+        urlPath: healthMonitor.urlPath,
+      };
+    }
+
+    return this.$http
+      .post(
+        `/cloud/project/${projectId}/region/${region}/loadbalancing/healthMonitor`,
+        postDatas,
+      )
+      .then(() => this.broadcast('useraccount.security.users.refresh', {}));
```

Comment:
Where is defined `broadcast` function ?

Reply:
wtf I don't know where this come from I remove it